core/src/main/java/org/apache/servicecomb/core/Invocation.java

```diff
@@ -188,11 +209,46 @@ public String getMicroserviceQualifiedName() {
     return operationMeta.getMicroserviceQualifiedName();
   }
 
+  protected void initTraceId() {
+    for (TraceIdGenerator traceIdGenerator : TRACE_ID_GENERATORS) {
+      initTraceId(traceIdGenerator);
```

Comment:
ok
customers should have a chance to override generator for the same traceId key
i will enhance the init logic.

Reply:
for generators have the same name, will only use the minimum order instance
   not use getTraceIdKeyName to control this logic, because most customers not want to generate multiple traceIds